tests/50federation/51transactions.pl

```diff
@@ -55,31 +55,62 @@
 # sent. It is unclear if this is the correct behavior or not.
```

Comment:
Could you update this comment as well? Perhaps say that the transaction should succeed, but it's unclear whether the response body should contain the invalid event.

Reply:
Done in https://github.com/matrix-org/sytest/commit/5359ab1401bbc114419068512f05b5c25d82a499 and in 842e5b71